cc/algorithms/algorithm.h

```diff
@@ -55,7 +55,10 @@ class Algorithm {
   // Epsilon is a standard parameter of differentially private
   // algorithms. See "The Algorithmic Foundations of Differential Privacy" p17.
   explicit Algorithm(double epsilon)
-      : epsilon_(epsilon), privacy_budget_(kFullPrivacyBudget) {}
+      : epsilon_(epsilon), delta_(kDefaultDelta), privacy_budget_(kFullPrivacyBudget) {}
```

Comment:
Can you call the other constructor from this constructor with delta set to 0?

Reply:
Fixed in the next commit.